package.json

```diff
@@ -35,11 +35,13 @@
     "babel-core": "5.8.33",
     "babel-eslint": "4.1.3",
     "babel-loader": "5.4.0",
-    "babel-plugin-react-transform": "1.1.1",
+    "babel-plugin-react-transform": "^1.1.1",
```

Comment:
Please don't unpin these. See ["Software Versions are Broken"](https://medium.com/javascript-scene/software-versions-are-broken-3d2dc0da0783#.ce8a3737u) for details.